easybuild/easyconfigs/p/pyke/pyke-1.1.1-intel-2016a-Python-3.5.1.eb

```diff
@@ -0,0 +1,27 @@
+easyblock = 'PythonPackage'
+
+name = 'pyke'
+version = '1.1.1'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'http://pyke.sourceforge.net/'
+description = """Pyke introduces a form of Logic Programming (inspired by Prolog) to the Python community by providing a knowledge-based inference engine (expert system) written in 100% Python."""
+
+toolchain = {'name': 'intel', 'version': '2016a'}
+
+#source_urls = [('https://sourceforge.net/projects/pyke/files/pyke/%(version)s/', 'download')],
+
+sources = ['pyke3-%(version)s.zip']
+source_urls = [('https://sourceforge.net/projects/pyke/files/', 'download')]
```

Comment:
again, 

    source_urls = [PYPI_SOURCE]
    sources = [SOURCE_TAR_GZ]

doesn't suceed in downloading; again on the PyPI page, there is no download link (well, the one there actually points to SF)